src/simularium/VisData.ts

```diff
@@ -78,13 +92,23 @@ class VisData {
         return this.frameCache.containsTime(time);
     }
 
+    public hasLocalCacheForFrame(frameNumber: number): boolean {
+        return this.frameCache.containsFrameAtFrameNumber(frameNumber);
+    }
+
     public gotoTime(time: number): void {
         const frameNumber = this.frameCache.getFrameAtTime(time)?.frameNumber;
         if (frameNumber !== undefined) {
             this.currentFrameNumber = frameNumber;
         }
     }
 
+    public gotoFrame(frameNumber: number): void {
```

Comment:
rename goToCachedFrame, future, figure out how to only check once: throw error error if not in cache